views/partials/addReview.ejs

```diff
@@ -0,0 +1,17 @@
+<script src="/js/addReviewClient.js" defer></script>
+
+<div class="add-review">
+    <button class="add-review-button">Write a review</button>
+    <div class="add-review-content">
+      <form>
+        <label for="name">Name:</label>
+        <input type="text" id="name" name="name" /><br />
+
+        <label for="rating">Rating:</label>
+        <input type="number" min="0" max="5" id="rating" name="rating" /><br />
+      </form>
+      <label for="comment">Write a comment: (Optional)</label><br />
+      <textarea cols="25" rows="4"></textarea><br />
+      <button class="submit-review">Submit review</button>
+    </div>
+  </div>
```

Comment:
The markup here is a bit strange. Why is the comment field outside of the `<form>`? And for that matter, the submit button?

Reply:
wops.. Honest mistake. I used "submit" earlier instead of "button", and when I changed to button I moved it outside the form. Will fix both comment and button! 👍